androidClient/app/src/main/java/com/macbitsgoa/comrades/coursematerial/CourseMaterial.java

```diff
@@ -67,13 +72,27 @@
     private String iconLink;
 
     @ColumnInfo(name = "filePath")
-    private String filePath;
+    public String filePath;
 
     @ColumnInfo(name = "fileSize")
     private Long fileSize;
 
-    @Ignore
```

Comment:
I didn't understand the purpose of changes on CourseMaterial and CourseActivty class.

Reply:
done